### ISSUE TITLE
Client Constructor does not list other params as optional

### ISSUE THREAD

**Issue by pikiaboy** [bitpay/nodejs-bitpay-client#50]
In Client.d.t.s, the constructor does not list the other params as optional. 

This leads to users not being able to only provide a path file as a means to create a new Client instance.

**Comment by svashish305** [comment]
@pikiaboy i faced a similar problem but i was able to instantiate client using configFilePath, but the other approach wherein it accepts other params, like token and pvt key, I got this error on create and get invoice methods:
`error Unable to get invoice due to:  failed to deserialize BitPay server response (Invoice) : RESTcli GET failed : 401 - {"error":"This endpoint does not support the `user` facade"}`

**Comment by anbugal** [comment]
Could you please describe this issue in more detail? 

the following example show how to initialise the client both ways

```node
        const {Client, Env, Currency, Models, Tokens, Facade} = require('bitpay-sdk');
        ...
        
        let tokens = Tokens;

        tokens.merchant = 'MERCHANT_TOKEN';

        let keyPlainText = 'PRIVATE_KEY';

        let client = new Client(null, Env.Test, keyPlainText, tokens);
        // OR
        let client = new Client("./BitPay.config.json");
```

**Comment by jekh** [comment]
Already fixed in #41

**Comment by anbugal** [comment]
Fixed on v3.0.2202
